page_analyzer/utils/db.py

```diff
@@ -0,0 +1,90 @@
+import os
+from typing import Any, Dict, List
+
+import psycopg2
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def connect_db() -> Any:
+    conn = psycopg2.connect(os.getenv("DATABASE_URL"))
+    return conn
+
+
+def get_url_context(url_id: int) -> Dict[str, str]:
+    conn = connect_db()
+    cur = conn.cursor()
+    cur.execute(
+        "SELECT id FROM urls WHERE id=%(int)s;",
+        {
+            "int": url_id,
+        },
+    )
+    target_id = cur.fetchone()
+    if target_id:
+        cur.execute(
+            "SELECT name, created_at FROM urls WHERE id=%(int)s;",
+            {
+                "int": url_id,
+            },
+        )
+        page_url, created_at = cur.fetchone()
+        cur.close()
+        conn.close()
+        context = {
+            "id": str(url_id),
+            "name": str(page_url),
+            "created_at": str(created_at),
+        }
+        return context
```

Comment:
I have a feeling that this function can be simplified
wdyt about returning actual DB data or None if such a URL doesn't exist?